spec.md

```diff
@@ -19,15 +19,209 @@
 
 ### Space Data Model
 
+`Space` properties:
+
+* `id` - deterministically set by server if not provided.
+* `type` - set by server to the string `Space`
```

Comment:
should we allow `['{uri}', 'Space']`? (imo yes)

(I'd also be willing to constrain and say these arrays must be sorted)

Reply:
ohhh right, arrays. Yeah one sec...

Reply:
Updated